js/main.js

```diff
@@ -7,55 +7,88 @@ var $defaultImage = document.querySelector('.default-image');
 
 var $url = document.querySelector('#journal-URL');
 
+var $title = document.querySelector('#journal-title');
+
+var $note = document.querySelector('#journal-notes');
+
+var $dataEntryId = data.nextEntryId;
```

Comment:
This should be removed, you don't need a second variable to keep track of the next id to use.